docs/SETUP.md

```diff
@@ -93,32 +93,9 @@ More details here: [React Native Android Setup](https://facebook.github.io/react
 
 ### Auth0
 
-#### Configuration
-
-If you don't want to use Auth0, or you want to take it into use later, you can skip this step for now.
-
-1. Before you start you need to create a new application in [Auth0](https://manage.auth0.com/#/applications/)
```

Comment:
Why are we not moving this part of documentation to the `Auth0.md` file instead of removing it?